### ISSUE TITLE
Error with DateTimeOffset

### ISSUE THREAD

**Issue by ljubomir-bacovic** [intuit/TSheets-V1-DotNET-SDK#13]
There is an error with requests that have DateTimeOffset parameter.

For example:
I am using CreateScheduleEvent method with ScheduleEvent defined in this way:
```
           var se = new ScheduleEvent
           {
               Active = true,
               AssignedUserIds = new[] {currentUser.Item1.Id},
               Color = ScheduleEventColor.Green,
               JobcodeId = jobCodeId,
               ScheduleCalendarId = c.Item1[0].Id,
               Start = DateTime.UtcNow.AddDays(3),
               End = DateTime.UtcNow.AddDays(3).AddHours(5)
           };
```

The exception I get is this:
"Invalid ISO-8601 value for start: [2019-11-14T16:11:18.4386423+00:00]"

I assume that helper class serializes the DateTimeOffset in a JSON format that is not acceptable to API.

Screenshot:
![image](https://user-images.githubusercontent.com/8394673/68602209-39dba680-04a6-11ea-9520-98263071e47a.png)

**Comment by dustinsweet** [comment]
Thanks for reporting this Ljubomir, I appreciate the detail - made for an easy repro.  I just pushed a fix to master, available in v1.2.1.  Please let us know if it's working now.

-Dustin

**Comment by ljubomir-bacovic** [comment]
Dustin, 

Great, it works now! 

Thanks,
Ljubomir
